assignments/assignment-8/assignment-8.3.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.3</title>
+    </head>
+    <body>
+        <h1>Job Application</h1>
+        <form>
+            <div>
+                <label for="fullNameInput">Full Name</label>
+                <input
+                    type="text"
+                    placeholder="Full Name"
+                    name="fullName"
+                    id="fullNameInput"
+                />
+
+                <label for="phoneNumberInput">Contact Number</label>
+                <input
+                    type="number"
+                    placeholder="Phone Number"
+                    name="phoneNumber"
+                    id="phoneNumberInput"
+                />
+
+                <label for="emailInput">Your Email</label>
+                <input
+                    type="email"
+                    placeholder="Email"
+                    name="email"
+                    id="emailInput"
+                />
+            </div>
+
+            <div>
+                <label for="coverLetterTextarea"
+                    >Please write your cover letter here</label
+                >
+                <textarea
+                    name="applicantCoverLetter"
+                    id="coverLetterTextarea"
+                    cols="30"
+                    rows="2"
+                ></textarea>
+
+                <label for="fileInput">Upload Your Resume in PDF format</label>
+                <input
+                    type="file"
+                    name="applicantResume"
+                    id="fileInput"
+                    accept="PDF"
+                />
+            </div>
+            <div>
+                <label for="levelEducationSelect"
+                    >Select your highest Level of Education</label
+                >
+                <select name="levelEducation" id="levelEducationSelect">
+                    <option value="Associate Degree">Associate Degree</option>
```

Comment:
assignment not include associate and doctor degree.

Reply:
they are just levels of education. I thought it's better then 1,2,3,4 levels